### PR TITLE
Houdini: fix wrong path in ASS loader

### PR DESCRIPTION
**Pull request by antirotor**
## Fix

ASS loader was filling in only file name, not the whole path. This is fixing the problem.

### Testing

Load ASS file with OpenPype in Houdini, everything should work as expected.

### CODE REVIEW

File: openpype/hosts/houdini/plugins/load/load_ass.py [hunk 1]
```diff
@@ -105,3 +67,23 @@ def remove(self, container):
...
+    @staticmethod
+    def format_path(path, representation):
+        """Format file path correctly for single bgeo or bgeo sequence."""
+        if not os.path.exists(path):
+            raise RuntimeError("Path does not exist: %s" % path)
+
+        is_sequence = bool(representation["context"].get("frame"))
+        # The path is either a single file or sequence in a folder.
+        if not is_sequence:
+            filename = path
```
Comment (iLLiCiTiT) [inline openpype/hosts/houdini/plugins/load/load_ass.py:80]: Ridiculous question: Is it a filename or filepath?

File: openpype/hosts/houdini/plugins/load/load_ass.py [hunk 2]
```diff
@@ -105,3 +67,23 @@ def remove(self, container):
...
+        else:
+            filename = re.sub(r"(.*)\.(\d+)\.(ass.*)", "\\1.$F4.\\3", path)
+
+            filename = os.path.join(path, filename)
```
Comment (iLLiCiTiT) [inline openpype/hosts/houdini/plugins/load/load_ass.py:84]: Shouldn't it be like this?
```suggestion
            filename = os.path.join(os.path.dirname(path), filename)
```

Comment (iLLiCiTiT) [reply openpype/hosts/houdini/plugins/load/load_ass.py:84]: Or to completelly simplify the function just
```
if is_sequence:
    dirpath, filename = os.path.split(path)
    new_filename = re.sub(r"(.*)\.(\d+)\.(ass.*)", "\\1.$F4.\\3", filename)
    path = os.path.join(dirpath, new_filename )

return os.path.normpath(path).replace("\\", "/")
```

**Review by iLLiCiTiT — APPROVED** [review]

**Comment by mkolar** [comment]
@antirotor maybe we could add this request into the PR? https://discord.com/channels/517362899170230292/1035450698655088661/1035450701045846026

**Pull request merged**